micro-rdk/Cargo.toml

```diff
@@ -86,6 +86,8 @@ pin-project.workspace = true
 async-lock.workspace = true
 dns-message-parser.workspace = true
 printf-compat = { workspace = true, optional = true }
+bincode = "2.0.0-rc.3"
+
```

Comment:
I can make this an optional dependency for `ota` feature moving forward. That way the changes it doesn't get into the regular build

will rm ws

Reply:
moved to just being compiled when target isn't esp32 for now. the [ticket](https://viam.atlassian.net/browse/RSDK-9342?atlOrigin=eyJpIjoiZDgzNGNlNTlmOTlhNGJlYTk3OTNiMjE3YzcwM2I3M2IiLCJwIjoiaiJ9) for having a native representation for the app description might use a different crate eventually so defering until then.